### ISSUE TITLE
Would it be possible to add color correction options to the plugin? (Request)

### ISSUE THREAD

**Issue by OuterWinnie** [libvibrant/vibrantDeck#1]
Hi! I was wondering is it would be possible to add color correction settings to the plugin, with the objetive of manage something like this but system wide:

https://www.reddit.com/r/SteamDeck/comments/wd2fa7/get_a_more_color_accurate_steam_deck_files/
https://www.reddit.com/r/SteamDeck/comments/v80w7u/steam_deck_display_calibration_v2/

Thanks for your work! Amazing plugin!

**Comment by Scrumplex** [comment]
I don't think you can set LUTs directly, as they are used in those posts. But color correction is definitely possible.

For future reference, these are all color values, that can be altered in Gamescope: https://github.com/Plagman/gamescope/blob/3122d1aabec4d8f7df79edb1b76e136c0dd5ffbf/src/steamcompmgr.cpp#L4937-L4942

**Comment by OuterWinnie** [comment]
Well, it would be amazing if you add those options at some point! Thanks again!

**Comment by aizakow** [comment]
I second this! Some way to match the calibration of those reshade configs would be amazing

**Comment by crystalclear23** [comment]
I third this.

**Comment by rc2189** [comment]
It seems like changing the gamma would be the next best addition:

`GAMESCOPE_COLOR_GAMMA_EXPONENT`

I tinkered around with it for a little while but never got it fully working

**Comment by rc2189** [comment]
@Scrumplex any chance you can add settings to change the gamma? In my opinion the deck could really benefit from those changes! I can collaborate if needed

**Comment by Scrumplex** [comment]
I have looked at the relevant code inside gamescope. That variables takes six float values. The first three are the exponents for degamma (I assume R, G and B) and the latter three are for gamma (also R, G and B).
Does anyone know what degamma is? :D

**Comment by rc2189** [comment]
> I have looked at the relevant code inside gamescope. That variables takes six float values. The first three are the exponents for degamma (I assume R, G and B) and the latter three are for gamma (also R, G and B). Does anyone know what degamma is? :D

I only dug through a bit, but that would make sense RGB then Gamma. Hopefully this is something that can be accomplished!

**Comment by rc2189** [comment]
I see a commit for this, huge thank you for considering adding the gamma! Hugely appreciated!

**Comment by crystalclear23** [comment]
How do you install the new version or do I just wait for it on decky?

**Comment by Scrumplex** [comment]
Gamma support is basically ready with PR #8.

I asked in Decky Discord for people to test it. If no issues surface, I will release a new version with gamma setting soon

**Comment by rc2189** [comment]
Fantastic, thank you!

**Comment by crystalclear23** [comment]
Is there a way to edit the index.js so that the increments are like 5 instead of 50

**Comment by aizakow** [comment]
The latest version appears to disable night mode whenever I launch a game, permanently until I go into system settings and turn it back on. Is this intended behavior? I sort of see the reasoning, but a toggle might be nice

**Comment by Scrumplex** [comment]
> Is there a way to edit the index.js so that the increments are like 5 instead of 50

I can't influence that sadly. If you use the touchscreen, you can set finer values

**Comment by Scrumplex** [comment]
> The latest version appears to disable night mode whenever I launch a game, permanently until I go into system settings and turn it back on. Is this intended behavior? I sort of see the reasoning, but a toggle might be nice

Steam also uses some way of editing gamma values. I am still not sure how they do it, but you (currently) can't have both
